plugins/main/ptp/ptp.go

```diff
@@ -57,93 +57,114 @@ func setupContainerVeth(netns, ifName string, mtu int, pr *types.Result) (string
 	// "192.168.3.1/32 dev $ifName" and "192.168.3.0/24 via 192.168.3.1 dev $ifName".
 	// In other words we force all traffic to ARP via the gateway except for GW itself.
 
-	var hostVethName string
+	var hostInterface, containerInterface *types.Interface
+
 	err := ns.WithNetNSPath(netns, func(hostNS ns.NetNS) error {
-		hostVeth, _, err := ip.SetupVeth(ifName, mtu, hostNS)
+		hostVeth, contVeth, err := ip.SetupVeth(ifName, "eth%d", mtu, hostNS)
```

Comment:
This is used as kernel template name so in this place this looks ok, but after adding random suffix it will look quite bad.
IMO there should be "eth" or "eth%d-".